pages/draft-04/migration-notes.md

```diff
@@ -0,0 +1,87 @@
+| `dependencies`    | `dependencies`    | Core          | Assertion         | The `dependencies` member values were changed to require an array of strings or a schema, eliminating the use of single strings.                                       |
+| `id`              | `id`              | Core          | Identifier        | -                                                                                                                                                                      |
+| Not present       | `allOf`           | Core          | Applicator        | -                                                                                                                                                                      |
+| Not present       | `anyOf`           | Core          | Applicator        | -                                                                                                                                                                      |
+| Not present       | `definitions`     | Validation    | Reserved Location | -                                                                                                                                                                      |
+| Not present       | `maxProperties`   | Validation    | Assertion         | -                                                                                                                                                                      |
+| Not present       | `minProperties`   | Validation    | Assertion         | -                                                                                                                                                                      |
+| Not present       | `not`             | Core          | Applicator        | -                                                                                                                                                                      |
+| Not present       | `oneOf`           | Core          | Applicator        | -                                                                                                                                                                      |
+
+#### Helpful notes for Keyword changelog
+
+1. `type`
+
+In Draft-03, the `type` keyword could hold a simple value like "string" or "number" and an entire schema as its value.
+
+For example:
+
+```json
+{
+  "type": {
+    "type": "array",
+    "items": { "type": "string" }
+  }
+}
+```
+
+This feature was in Draft-03 but changed in later versions of JSON Schema, where `type` is limited to simpler values like strings or arrays of strings.
+
+The `allOf` keyword has now replaced the previous type schema functionality.
+
+<Infobox label="Note"> Starting with Draft 4, schema identifiers that use an empty URI "" or a fragment-only URI "#" are no longer allowed.
+
+In Draft 3, these identifiers were considered valid:
+
+```json
+id: ""
```

Comment:
I think you should enclose `id` on double quotes to make this valid JSON

Reply:
Thank you, updated.